cmd/migrations/init.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func Run() {
+	c := &cobra.Command{
+		Short:        "Used to repair broken swarm references",
```

Comment:
that's true just for the manifest specific stuff in repair dir and file. also, i wouldn't use "broken" :) but deprecated